README.md

```diff
@@ -26,7 +26,7 @@ pip install mintapi
 
 From the command line, the most automated invocation will be:
 
-    python mintapi/api.py --keyring --headless you@example.com
+    mintapi --keyring --headless you@example.com
```

Comment:
Does this still need the python binary, or am I missing how this would work with just a plain 'mintapi'? Overall looks awesome, this was the only part I was unclear on. Thanks for this undertaking!

Reply:
It works both ways with the caveat that if you put the python binary there, you need to have a local repo of mintapi in your current working directory as python will treat it like a path. I think that will be an abnormal situation as most will install mintapi from PyPI. 

If you don't put the python binary there, your terminal process will search on your `PATH` to find mintapi and it will be there if you have mintapi installed globally or in an activated virtual environment. This is the way that we already recommend as you can see from the CLI section at the end of the README.md.

Reply:
Ah cool, I had forgotten that, thanks! Makes sense.